test/test_exporter.py

```diff
@@ -571,6 +571,37 @@ def test_configuration_custom(self):
                 self.assertIsInstance(kwargs.get("exporter"),
                                       _DynatraceMetricsExporter)
 
+    @patch.object(requests.Session, 'post')
+    def test_nonstring_attributes_dropped(self, mock_post):
```

Comment:
Could we make the test "generic", meaning we pass 2 params to it (type of instrument + expected string line)? Then we would be sure that the attributes exclusion is done right for all metric types. That would catch if in the future the code is changed to not use the filtering function.

Reply:
Thanks for effort in the tests!